tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -833,11 +854,11 @@ def clean_up(self, peercluster, **kw):
                 self.delete_pool(poolname=pool)
                 peercluster.delete_pool(poolname=pool)
 
-    def get_rbd_service_name(self, service_name):
+    def get_rbd_service_name(self, service_name="rbd-mirror"):
```

Comment:
Which other service names are we currently fetching using this method?

Reply:
just rbd-mirror

Reply:
okay. i think we can leave it like this for now.